src/components/A/A.scss

```diff
@@ -1,7 +1,5 @@
 .A {
   .Icon {
     display: inline-flex;
-    margin-left: 1px;
```

Comment:
hey @angle943, I have implemented based on the new criterias. However, since there were margins added on your previous commit, it was interfering with the margins in the `TopNavPopoverItem` components, basically your newly added `.A .Icon` styles has a higher specificity. 

<img width="973" alt="Screen Shot 2021-02-14 at 10 33 24 PM" src="https://user-images.githubusercontent.com/32864116/107879588-b1ec1400-6f14-11eb-84b8-e66bfac495f2.png">

I removed it for now, as it was not impacting the existing UI much? But would be open if there's a better way to maintain both UIs.

Reply:
Update: 
I added the margins back to the `.A .Icon` styles, however, I made it more specific by adding the `--new-window` modifier.

```
A {
   // property must be placed in `.Icon` instead of `&__Icon--new-window` to 
   // prevent original `.Icon` display flex style to override this
  .Icon {
    display: inline-flex;
  }

  &__Icon--new-window {
    margin-left: 1px;
    margin-right: 3px;
  }
}
```

Reply:
ah that `.Icon{...}` was an oversight in my part. Shouldn't have made the styling that way